pkg/apis/pipeline/v1alpha1/resource_types.go

```diff
@@ -140,6 +142,12 @@ type TaskResourceBinding struct {
 	Paths []string `json:"paths,omitempty"`
 }
 
+// PipelineResourceResult used to export the image name and digest as json
+type PipelineResourceResult struct {
+	Name   string `json:"name"`
+	Digest string `json:"digest"`
```

Comment:
hmm `Digest` seems very specific to the image type of `PipelineResource`, but theoretically other types of `PipelineResources` might want to add data here - any thoughts on how we could make this more flexible? (maybe embedding freeform json isnt the worst idea? or could duck typing help?)

Reply:
I was assuming other types could add their own fields as they need to in the same type, wdyt?

Reply:
Sure we can stick with that for now